packages/cli/src/commands/generate/cell/cell.js

```diff
@@ -61,9 +63,30 @@ export const files = async ({
     templateNameSuffix = 'List'
     // override operationName so that its find_operationName
   }
-  const operationName = await uniqueOperationName(cellName, {
-    list: shouldGenerateList,
-  })
+
+  let operationName = options.query == null ? '' : options.query.trim()
+  if (operationName == '') {
+    operationName = await uniqueOperationName(cellName, {
+      list: shouldGenerateList,
+    })
+  } else {
+    const userSpecifiedNameIsUnique = await operationNameIsUnique(operationName)
+    if (!userSpecifiedNameIsUnique) {
```

Comment:
Shall we remove the prompt here and just error out?

Reply:
Yes best to remove the prompt until the behaviour is better supported. See 26743639e71b14830ecde0bfaaf64df832bb996c for change.

Reply:
I think this is now taken on board.